### PR TITLE
[ZH] Add `Beatmap\Technical_maps`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmap/Technical_maps/zh.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  - slider
+  - SV
+  - term
+  - 作图
+  - 制谱
+  - 科技
+  - 技术
+  - 谱面
+  - 滑条
+  - 术语
```
Comment (kj415j45) [inline wiki/Beatmap/Technical_maps/zh.md:15]:
```suggestion
  - 术语
  - 变速
  - 滑条速度
```

File: wiki/Beatmap/Technical_maps/zh.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+---
+
+# 科技谱
+
+<!-- 最后添加一行：*请勿与[滑条艺术](link)或[Taikosu 谱](link)（一种优先考虑太鼓转谱游玩的主模式谱面）相混淆。 -->
+
+*注意：在本文中，“科技谱”是包含以下所有定义的总括术语。*
+
+**科技谱**（常简称为 “tech map”） 是一个表示谱面暂未被定义为一个 osu! 社区玩家意见一致、特点清晰的谱面风格的专有名词。在玩家群体中，“科技谱”也常用于概括一类仅仅包含特殊[排列](/wiki/Beatmap/Pattern)的[谱面](/wiki/Beatmap)，或是一类[物件](/wiki/Hit_object)密集，包含多种特殊的[滑条形状](/wiki/Hit_object/Slider)且滑条速度经常大幅度变化的谱面。
```
Comment (kj415j45) [inline wiki/Beatmap/Technical_maps/zh.md:24]:
```suggestion
**科技谱**（常简称为 “tech map”） 这个专有名词目前并未被 osu! 社区明确而统一地定义。在玩家群体中，“科技谱”可用于概括一类仅仅包含特殊[排列](/wiki/Beatmap/Pattern)的[谱面](/wiki/Beatmap)，或是一类[物件](/wiki/Hit_object)密集，包含多种特殊的[滑条形状](/wiki/Hit_object/Slider)且滑条速度经常大幅度变化的谱面。
```

File: wiki/Beatmap/Technical_maps/zh.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+科技谱经常含有结构复杂、节奏丰富的歌曲（如：DnB、Dubstep、Drumstep 风格）。这种歌曲让谱师能够捕捉歌曲内丰富的声音从而制作更复杂、花样繁多的科技谱。当然，科技谱内的歌曲也不一定是这种风格。
```
Comment (kj415j45) [inline wiki/Beatmap/Technical_maps/zh.md:26]:
```suggestion
科技谱往往与结构复杂、节奏丰富的歌曲（如：DnB、Dubstep、Drumstep 风格）密切相关。因为这种歌曲让谱师能够捕捉歌曲内每一个声音，从而制作更复杂、花样繁多的科技谱。当然，科技谱的歌曲也不一定是这种风格。
```

File: wiki/Beatmap/Technical_maps/zh.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+但不管定义如何，许多玩家抱怨科技谱因为含有迷惑排列和特殊滑条形状而导致谱面实际难度与星数不匹配，游玩这种图也需要玩家拥有更高的实力水平。同样地，这也意味着当前星数系统低估了科技谱的真实难度，不能因为能够轻松获得大量[pp](/wiki/Performance_points)而使得科技谱在玩家群体中流行。
```
Comment (kj415j45) [inline wiki/Beatmap/Technical_maps/zh.md:28]:
```suggestion
但不管定义如何，许多玩家抱怨科技谱因为含有迷惑排列和特殊滑条形状而导致谱面实际难度与星数不匹配，游玩这种图也需要玩家拥有更高的实力水平。同样地，这也意味着当前星数系统低估了科技谱的真实难度，玩家无法一次获得大量[pp](/wiki/Performance_points)而导致它们不太热门。
```

File: wiki/Beatmap/Technical_maps/zh.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+*注意：只需要谱面符合以下列表内一到两个特点，即可广泛定义为科技谱。*
+
+- 非常规或难读的谱面排列（如以下的例子）
+  - 含有很“硬”的 flow，需要较强的手控能力游玩的谱面
+- 变速滑条
+  - 或包含艺术滑条
+- 特殊滑条形状
+- 滑条速度突变
+- 高密度的物件
+  - 谱面大部分时间都被高密度物件所覆盖（出现大量[连打](/wiki/Beatmap/Pattern/Stream)类排列）
```
Comment (kj415j45) [inline wiki/Beatmap/Technical_maps/zh.md:45]:
```suggestion
  - 谱面大部分时间都被高密度物件所覆盖（但不是[连打](/wiki/Beatmap/Pattern/Stream)）
```
Exclude 是“除了……之外”……

**Review by kj415j45 — CHANGES_REQUESTED** [review]

**Review by kj415j45 — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**